app/src/main/java/practicum/getfitla_v3/Nutrition_New_Page.java

```diff
@@ -0,0 +1,38 @@
+package practicum.getfitla_v3;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+/**
+ * Created by Work on 3/17/18.
+ */
+
+public class Nutrition_New_Page extends AppCompatActivity{
```

Comment:
Also, I'm concerned that this naming convention is too similar to "Nutrition.java." You should either add a block comment explaining the difference or choose a more intuitive name.

Reply:
Something that ends with "Activity" is likely appropriate. Also, keep to the camelCase convention instead of the Mixed_Camel_Snake_Case